hack/dev-build.sh

```diff
@@ -45,6 +45,7 @@ KOPS_DIRECTORY="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 [ -z "$KOPS_STATE_STORE" ] && echo "Need to set KOPS_STATE_STORE" && exit 1;
 [ -z "$CLUSTER_NAME" ] && echo "Need to set CLUSTER_NAME" && exit 1;
 [ -z "$NODEUP_BUCKET" ] && echo "Need to set NODEUP_BUCKET" && exit 1;
+[ -z "$IMAGE" ] && echo "Need to set IMAGE or use the image list https://github.com/kubernetes/kops/blob/master/channels/stable" && exit 1;
```

Comment:
I re-read this and I am confused.  I would like to force the user to specify an image, be the user is a developer.  Thoughts?